File: crates/local-review-core/tests/semantic-golden/python/01-functions.py
def authenticate(user, password):
    return user == "admin" and password == "secret"


def validate_token(token):
    return bool(token)


class Session:
    def __init__(self, session_id):
        self.session_id = session_id

    def refresh(self):
        return bool(self.session_id)

File: crates/local-review-core/tests/semantic-golden/rust/01-functions.rs
pub fn authenticate(user: &str, pass: &str) -> bool {
    user == "admin" && pass == "secret"
}

pub fn validate_token(token: &str) -> bool {
    !token.is_empty()
}

struct Session {
    id: String,
}

impl Session {
    pub fn new(id: String) -> Self {
        Self { id }
    }

    pub fn refresh(&self) -> bool {
        !self.id.is_empty()
    }
}
